docs/index.html

```diff
@@ -1711,81 +1711,81 @@
     });
 });
 </script>
-<style>

```

Comment:
There is no need to modify the generated index.html. Can you revert this change?